content/news/2024-09-20-esg-byos-im/index.md

```diff
@@ -0,0 +1,137 @@
+
+<img class="center" src="minio-create-bucket.png" width="80%" alt="Create a bucket in MinIO.">
+
+The user can then add this bucket to Galaxy by following the steps described at the beginning of this post (Select `Any S3 Compatible Storage` option from the available templates). The user needs to provide a `Name`, `Access Key ID`, `Bucket` (name of the bucket), the `S3-Compatible API Endpoint` (MinIO API endpoint), the `Secret Access Key` and submit the form.
+
+<img class="center" src="byos-add-minio-bucket-to-galaxy.png" width="80%" alt="Add your MinIO bucket to Galaxy.">
+
+Once the bucket is added to Galaxy the user can start using it as the default storage location or select it at different levels: per `History`, per `Tool`, and `Workflow`.
+
+<img class="center" src="byos-your-newly-added-external-storage.png" width="80%" alt="Newly added external storage.">
+
+# Running workflows in Galaxy with Bring Your Own Storage
+
+The user can now run tools and workflows in Galaxy using the newly configured external storage. Below is an example of a workflow that performs the assembly of bacterial genomes using [Shovill](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fshovill%2Fshovill%2F1.1.0%20galaxy2) a faster _de novo_ assembly pipeline for Illumina paired end reads.
+
+<iframe title="Galaxy Workflow Embed" style="width: 100%; height: 700px; border: none;" src="https://usegalaxy.eu/published/workflow?id=b17603457e1d4262&embed=true&buttons=true&about=true&heading=true&minimap=true&zoom_controls=true&initialX=-20&initialY=-20&zoom=1"></iframe>
+
+## Steps to run the workflow:
+1. Create a new history in Galaxy.
+2. Select the newly added external storage as the preferred storage location for the `History`.
+
+    <img class="center" src="byos-history-level-selection.png" width="80%" alt="Set storage for this history.">
+
+3. Upload / Import the [input data](https://usegalaxy.eu/published/history?id=8ac959df218c9188).
+4. Go to the `Workflow` tab and select the workflow and run it.
+
+    <img class="center" src="run-galaxy-workflow.png" width="80%" alt="Set storage for this history.">
+
+5. Once the workflow is completed,
+
+    <img class="center" src="successful-completion-of-the-workflow-invocation.png" width="80%" alt="Successful completion of workflow run">
+
+    the user can find all the output files in the external storage.
+
+    <img class="center" src="all-workflow-datasets-now-in-the-bucket.png" width="80%" alt="Output files in external storage.">
+
+6. The user can also evaluate it by clicking the `Dataset details` of a job and observe that the user's external storage is used as the `Dataset Storage`
+
+    <img class="center" src="dataset-storage-location.png" width="80%" alt="Dataset details showing external storage.">
```

Comment:
Can you please ack the original PR at the end?

Reply:
Added now.